apps/web/src/hooks/useNotificationInfo.ts

```diff
@@ -222,6 +223,31 @@ const userFileGeneratedNotificationToInfo = (
   };
 };
 
+function isVerifyWorkEmailNotification(
+  notification: GraphqlType,
+): notification is VerifyWorkEmailNotification {
+  return notification.__typename === "VerifyWorkEmailNotification";
+}
+
+const verifyWorkEmailNotificationToInfo = (
+  paths: ReturnType<typeof useRoutes>,
+  intl: IntlShape,
+): NotificationInfo => {
+  const message = intl.formatMessage({
+    defaultMessage:
+      "According to your profile, you currently work for the Government of Canada. Gain access to employee-only tools by verifying your work email.",
+    id: "mbQw3Z",
+    description:
+      "Notification suggesting to a user that they should verify a work email",
+  });
+
+  return {
+    message: message,
+    label: message,
+    href: `${paths.profile()}#government-section`,
```

Comment:
Instead of static, you could make this dynamic in case it changes if you imported `PAGE_SECTION_ID.GOVERNMENT` from `apps/web/src/constants/sections/userProfile.ts` and then:

```
href: `${paths.home()}#${PAGE_SECTION_ID.GOVERNMENT}`,
```